src/apps/ENGINE/crashpad.cpp

```diff
@@ -0,0 +1,42 @@
+#include "crashpad.hpp"
+
+#include "fs.h"
+#include "vfile_service.h"
+
+#include <client/crash_report_database.h>
+#include <client/crashpad_client.h>
+#include <client/settings.h>
+
+namespace storm::except
+{
+
+bool startCrashHandler()
```

Comment:
Most of the codebase uses CamelCase style. Probably we should stick to it.

Reply:
Our coding style describes exactly this naming...

Reply:
I know, I left a comment there in this regard.

Reply:
as discussed, let's leave it as is right now as global reformat incoming